cdap-ui/app/hydrator/controllers/create/partials/nodeconfig-ctrl.js

```diff
@@ -447,102 +447,26 @@ class HydratorPlusPlusNodeConfigCtrl {
     }
     return true;
   }
+
   validatePluginProperties() {
     const nodeInfo = this.state.node;
     let vm = this;
-    vm.validating = true;
-    const pluginInfo = angular.copy(nodeInfo.plugin);
-    pluginInfo.type = nodeInfo.type;
-
-    const requestBody = {
-      stage: {
-        name: this.myHelpers.objectQuery(nodeInfo, 'name'),
-        plugin: pluginInfo
-      },
-      inputSchemas: !nodeInfo.inputSchema ? [] : nodeInfo.inputSchema.map(input => {
-        let schema;
-        try {
-          schema = JSON.parse(input.schema);
-        } catch (e) {
-          // no-op
-        }
-
-        return {
-          stage: this.myHelpers.objectQuery(input, 'name'),
-          schema,
-        };
-      })
-    };
-
-    const parseResSchema = (res) => {
-      if (res.name && res.type && res.fields) {
-        return [this.HydratorPlusPlusNodeService.getOutputSchemaObj(res)];
+    const errorCb = ({ errorCount, propertyErrors, inputSchemaErrors, outputSchemaErrors }) => {
+      // errorCount can be 0, a positive integer, or undefined (in case of an error thrown)
+      if ( errorCount > 0 ){
+        vm.propertyErrors = propertyErrors;
+        vm.inputSchemaErrors = inputSchemaErrors;
+        vm.outputSchemaErrors = outputSchemaErrors;
+      } else if ( errorCount === 0 ){
+        // Empty existing errors
+        vm.propertyErrors = [];
```

Comment:
this should be `vm.propertyErrors = {}`. I had this change in one of my PRs that's not merged yet. Can you make this change in other places where you're clearing propertyErrors?